src/test/scripts/generate-proof.js

```diff
@@ -0,0 +1,48 @@
+const ethers = require("ethers");
+const { ZkIdentity, Strategy } = require("@zk-kit/identity");
+const { Semaphore, generateMerkleProof } = require("@zk-kit/protocols");
+
+async function main() {
+	const airdropAddress = process.argv[2];
+	const signal = Buffer.from(process.argv[3].slice(2), "hex").toString("utf-8");
+
+	const identity = new ZkIdentity(Strategy.MESSAGE, "test-identity");
+	const identityCommitments = [
+		BigInt(1),
+		identity.genIdentityCommitment(),
+		BigInt(2),
+	];
+
+	const merkleProof = generateMerkleProof(
+		20,
+		BigInt(0),
+		identityCommitments,
+		1
+	);
+
+	const witness = Semaphore.genWitness(
+		identity.getTrapdoor(),
+		identity.getNullifier(),
+		merkleProof,
+		airdropAddress,
+		"fake-signal" //signal
```

Comment:
Using the real signal here causes an error, because the string is bigger than 31 bytes. [Here's the check that throws](https://github.com/ethers-io/ethers.js/blob/master/packages/strings/src.ts/bytes32.ts#L15), which happens when hashing the signal.

Reply:
fixed now :)